packages/common/src/shared-types.ts

```diff
@@ -303,6 +303,11 @@ export interface AdminUiConfig {
      * @default false
      */
     hideVersion?: boolean;
+    /**
+     * @description
+     * The custom login image
+     */
```

Comment:
There should be `@since 1.9.0` here to show in the docs in what version this option was added.